app/styles/ui/_branches.scss

```diff
@@ -14,6 +14,18 @@
     width: 365px;
     min-height: 0;
   }
+
+  .list-item.dragged-over {
+    --text-color: var(--box-selected-active-text-color);
+    --text-secondary-color: var(--box-selected-active-text-color);
+
+    color: var(--text-color);
+    background-color: var(--box-selected-active-background-color);
+  }
+
+  .branches-list-item {
+    pointer-events: none;
```

Comment:
This will likely have to change... If I did not have this, the drop area for the commit would only be the border around the branches-list-item on of action list item that contains it. The goal is to have the whole list item (containing list item, and branches-list-item) be the dropzone. Which this accomplishes.. but it also prevents any other pointer events... i.e., you can't invoke context menu.. 

But I would like to go ahead and bring in base drag and drop functionality before I head down the path of figuring out how to have the branch-list-item be a dropzone and behave the exact same way as the whole parent list item... it has issues with highlighting and obtaining the whole branch object.